searchworks-dev/solrconfig.xml

```diff
@@ -1043,6 +1043,7 @@
         managed_purl_urls,
         characteristics_ssim,
         author_struct:[json],
+        hathitrust_info_struct:[json],
```

Comment:
I _think_ this should be how this works (but not 💯 on that)

Reply:
LGTM. If we're going to have Solr decode JSON for the index page, should we also do it for the show page (down on 1067 or so?)

Reply:
I suppose.  😆